docs/2_2_common_schema.adoc

```diff
@@ -120,8 +120,6 @@ _[Note that the definition of an encoding scheme is a prerequisite in order for
 _Furthermore, note the FMI calling interface requires that strings are encoded in UTF-8._
 _Since the XML files are also required to be encoded in UTF-8, string variables need not to be transformed when reading from the XML files in to C string variables.]._
 
-The special values `NaN`, `+INF`, `-INF` for variables values are not allowed in the FMI XML files.
```

Comment:
... and, I think this should be an extra PR, as it does not fit well to this PR´s title and could confuse reviewers.